networkx/algorithms/centrality/voterank_alg.py

```diff
@@ -27,6 +25,32 @@ def voterank(G, number_of_nodes=None):
         Ordered list of computed seeds.
         Only nodes with positive number of votes are returned.
 
+    Examples
+    --------
+    >>> G= nx.Graph()
+    >>> G.add_edge(0,1)
+    >>> G.add_edge(0,2)
+    >>> G.add_edge(0,3)
+    >>> G.add_edge(1,4)
```

Comment:
IMO it's nice to have the example be as concise as possible so that the reader can focus on the relevant parts (e.g. `voterank` in this case).

```suggestion
    >>> G = nx.Graph([(0, 1), (0, 2), (0, 3), (1, 4)])
```